gammapy/makers/background/reflected.py

```diff
@@ -59,11 +59,11 @@ class ReflectedRegionsFinder:
     >>> on_region = CircleSkyRegion(target_position, theta)
     >>> finder = ReflectedRegionsFinder(min_distance_input='1 rad', region=on_region, center=pointing)
     >>> regions = finder.run()
-    >>> print(regions[0])
+    >>> print(regions[0]) # doctest: +SKIP
```

Comment:
Don't need to skip. You can use an ellipsis:
` # doctest: +ELLIPSIS`

and then in the result:

```
radius: 1438.3... arcsec
```